<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="ADM108_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ADM 108</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ADM204_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ADM 204</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ADM212_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ADM 212</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ADM220_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ADM 220</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ADM305_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ADM 305</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ADM324_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ADM 324</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BSB208_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BSB 208</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BSB214_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BSB 214</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BSB255_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BSB 255</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BSB289_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BSB 289</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DMC119_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DMC 119</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DMC142_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DMC 142</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="EDB102_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>EDB 102</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="EDB116_1C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>EDB 116</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FSB237_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FSB 237</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HWC156_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HWC 156</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HWC234_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HWC 234</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LIB267_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LIB 267</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MBB150_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MBB 150</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MCG25_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MCG 25</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MCG42_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MCG 42</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MCG43_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MCG 43</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MCK126_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MCK 126</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MCK142_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MCK 142</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MCK28_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MCK 28</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MCK3_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MCK 3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="WPA120_1C">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>WPA 120</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ZL201_1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ZEL 300</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ZE300_1H">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ZEL 201</string>
    </property>
   </widget>
   <widget class="QPushButton" name="installButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Install</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
